gum/arch-x86/gumx86writer.c

```diff
@@ -784,42 +786,15 @@ gum_x86_writer_put_jmp_near_ptr (GumX86Writer * self,
 }
 
 void
-gum_x86_writer_put_jcc (GumX86Writer * self,
-                        guint8 opcode,
-                        gconstpointer target,
-                        GumBranchHint hint)
-{
-  gsize short_instruction_size = 2, near_instruction_size = 6;
-  gint64 distance;
-
-  if (hint != GUM_NO_HINT)
-  {
-    short_instruction_size++;
-    near_instruction_size++;
-  }
-
-  distance = (gssize) target - (gssize) (self->pc + short_instruction_size);
-
-  if (GUM_IS_WITHIN_INT8_RANGE (distance))
-  {
-    gum_x86_writer_put_jcc_short (self, opcode, target, hint);
-  }
-  else
-  {
-    distance = (gssize) target - (gssize) (self->pc + near_instruction_size);
-    g_assert (GUM_IS_WITHIN_INT32_RANGE (distance));
-
-    gum_x86_writer_put_jcc_near (self, opcode, target, hint);
-  }
-}
-
-void
 gum_x86_writer_put_jcc_short (GumX86Writer * self,
-                              guint8 opcode,
+                              x86_insn instruction_id,
                               gconstpointer target,
                               GumBranchHint hint)
 {
   gssize distance;
+  guint8 opcode;
```

Comment:
Variables should be declared in the order that they are used, except if one of them represents the return value/result, in which case it goes first.